### PR TITLE
Add docs for 6.0 drawing public APIs

### PR DESCRIPTION
**Pull request by maryamariyan**
Fixes https://github.com/dotnet/runtime/issues/58653

### CODE REVIEW

File: xml/System.Drawing/PointF.xml [hunk 1]
```diff
@@ -680,9 +680,9 @@
         <Parameter Name="point" Type="System.Drawing.PointF" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="point">To be added.</param>
+        <param name="point">To point to convert from.</param>
```
Comment (maryamariyan) [inline xml/System.Drawing/PointF.xml:683]:
```suggestion
        <param name="point">The point to convert from.</param>
```

File: xml/System.Drawing/SizeF.xml [hunk 2]
```diff
@@ -795,9 +795,9 @@
         <Parameter Name="size" Type="System.Drawing.SizeF" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="size">To be added.</param>
+        <param name="size">Input of type <see cref="T:System.Drawing.SizeF" />.</param>
         <summary>Converts the specified <see cref="T:System.Drawing.SizeF" /> to a <see cref="T:System.Numerics.Vector2" />.</summary>
-        <returns>To be added.</returns>
+        <returns>Instance of type <see cref="T:System.Drawing.SizeF" />.</returns>
```
Comment (carlossanlop) [inline xml/System.Drawing/SizeF.xml:800]: The return value is incorrect:

```suggestion
        <returns>The size, converted into a <see cref="T:System.Numerics.Vector2" /> instance.</returns>
```

File: xml/System.Drawing/SizeF.xml [hunk 3]
```diff
@@ -823,9 +823,9 @@
         <Parameter Name="vector" Type="System.Numerics.Vector2" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="vector">To be added.</param>
+        <param name="vector">The source vector.</param>
         <summary>Converts the specified <see cref="T:System.Numerics.Vector2" /> to a <see cref="T:System.Drawing.SizeF" />.</summary>
-        <returns>To be added.</returns>
+        <returns>Instance of type <see cref="T:System.Drawing.SizeF" />.</returns>
```
Comment (gewarren) [inline xml/System.Drawing/SizeF.xml:828]:
```suggestion
        <returns>The pair of numbers that represents the converted <see cref="T:System.Numerics.Vector2" />.</returns>
```

Comment (carlossanlop) [inline xml/System.Drawing/SizeF.xml:828]:
```suggestion
        <returns>The vector, converted into a <see cref="T:System.Drawing.SizeF" /> instance.</returns>
```

File: xml/System.Drawing/SizeF.xml [hunk 4]
```diff
@@ -1193,7 +1193,7 @@
       <Parameters />
       <Docs>
         <summary>Creates a new <see cref="T:System.Numerics.Vector2" /> from this <see cref="T:System.Drawing.SizeF" />.</summary>
-        <returns>To be added.</returns>
+        <returns>Instance of type <see cref="T:System.Numerics.Vector2" />.</returns>
```
Comment (carlossanlop) [inline xml/System.Drawing/SizeF.xml:1196]:
```suggestion
        <returns>The size, converted into a new <see cref="T:System.Numerics.Vector2" /> instance.</returns>
```

**Review by carlossanlop — COMMENTED** [review]
Thanks. Left some suggestions for you to consider.

File: xml/System.Drawing/SizeF.xml (hunk 4 above)
Comment (gewarren) [inline xml/System.Drawing/SizeF.xml:1196]:
```suggestion
        <returns>The vector that was created from this <see cref="T:System.Drawing.SizeF" />.</returns>
```

File: xml/System.Drawing/RectangleF.xml [hunk 5]
```diff
@@ -1377,9 +1377,9 @@
         <Parameter Name="vector" Type="System.Numerics.Vector4" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="vector">To be added.</param>
+        <param name="vector">The source vector.</param>
```
Comment (gewarren) [inline xml/System.Drawing/RectangleF.xml:1380]:
```suggestion
        <param name="vector">The vector to convert.</param>
```

File: xml/System.Drawing/RectangleF.xml [hunk 6]
```diff
@@ -1377,9 +1377,9 @@
...
         <summary>Converts the specified <see cref="T:System.Numerics.Vector2" /> to a <see cref="T:System.Drawing.RectangleF" />.</summary>
-        <returns>To be added.</returns>
+        <returns>The new <see cref="T:System.Drawing.RectangleF" /> that this method creates.</returns>
```
Comment (gewarren) [inline xml/System.Drawing/RectangleF.xml:1382]:
```suggestion
        <returns>The rectangle that represents the converted <see cref="T:System.Numerics.Vector4" />.</returns>
```

File: xml/System.Drawing/RectangleF.xml [hunk 7]
```diff
@@ -1377,9 +1377,9 @@
         <Parameter Name="vector" Type="System.Numerics.Vector4" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="vector">To be added.</param>
+        <param name="vector">The source vector.</param>
         <summary>Converts the specified <see cref="T:System.Numerics.Vector2" /> to a <see cref="T:System.Drawing.RectangleF" />.</summary>
```
Comment (gewarren) [inline xml/System.Drawing/RectangleF.xml:1381]:
```suggestion
        <summary>Converts the specified <see cref="T:System.Numerics.Vector4" /> to a <see cref="T:System.Drawing.RectangleF" />.</summary>
```

File: xml/System.Drawing/RectangleF.xml [hunk 8]
```diff
@@ -1349,9 +1349,9 @@
         <Parameter Name="rectangle" Type="System.Drawing.RectangleF" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="rectangle">To be added.</param>
+        <param name="rectangle">The source rectangle.</param>
```
Comment (gewarren) [inline xml/System.Drawing/RectangleF.xml:1352]:
```suggestion
        <param name="rectangle">The rectangle to convert.</param>
```

File: xml/System.Drawing/RectangleF.xml [hunk 9]
```diff
@@ -1349,9 +1349,9 @@
...
         <summary>Converts the specified <see cref="T:System.Drawing.RectangleF" /> to a <see cref="T:System.Numerics.Vector4" />.</summary>
-        <returns>To be added.</returns>
+        <returns>The new <see cref="T:System.Numerics.Vector4" /> that this method creates.</returns>
```
Comment (gewarren) [inline xml/System.Drawing/RectangleF.xml:1354]:
```suggestion
        <returns>The vector that represents the converted <see cref="T:System.Drawing.RectangleF" />.</returns>
```

File: xml/System.Drawing/PointF.xml [hunk 10]
```diff
@@ -1073,7 +1073,7 @@
       <Parameters />
       <Docs>
         <summary>Creates a new <see cref="T:System.Numerics.Vector2" /> from this <see cref="T:System.Drawing.PointF" />.</summary>
-        <returns>To be added.</returns>
+        <returns>The new <see cref="T:System.Drawing.PointF" /> that this method creates.</returns>
```
Comment (gewarren) [inline xml/System.Drawing/PointF.xml:1076]:
```suggestion
        <returns>The vector that was created from this <see cref="T:System.Drawing.PointF" />.</returns>
```

File: xml/System.Drawing/PointF.xml [hunk 11]
```diff
@@ -708,9 +708,9 @@
         <Parameter Name="vector" Type="System.Numerics.Vector2" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="vector">To be added.</param>
+        <param name="vector">The source vector.</param>
```
Comment (gewarren) [inline xml/System.Drawing/PointF.xml:711]:
```suggestion
        <param name="vector">The vector to convert.</param>
```

File: xml/System.Drawing/PointF.xml [hunk 12]
```diff
@@ -708,9 +708,9 @@
...
         <summary>Converts the specified <see cref="T:System.Numerics.Vector2" /> to a <see cref="T:System.Drawing.PointF" />.</summary>
-        <returns>To be added.</returns>
+        <returns>The new <see cref="T:System.Drawing.PointF" /> that this method creates.</returns>
```
Comment (gewarren) [inline xml/System.Drawing/PointF.xml:713]:
```suggestion
        <returns>The pair of coordinates that represents the converted <see cref="T:System.Numerics.Vector2" />.</returns>
```

File: xml/System.Drawing/PointF.xml [hunk 13]
```diff
@@ -680,9 +680,9 @@
         <Parameter Name="point" Type="System.Drawing.PointF" Index="0" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="point">To be added.</param>
+        <param name="point">The point to convert from.</param>
         <summary>Converts the specified <see cref="T:System.Drawing.PointF" /> to a <see cref="T:System.Numerics.Vector2" />.</summary>
-        <returns>To be added.</returns>
+        <returns>The new <see cref="T:System.Numerics.Vector2" /> that this method creates.</returns>
```
Comment (gewarren) [inline xml/System.Drawing/PointF.xml:685]:
```suggestion
        <returns>The vector that represents the converted <see cref="T:System.Drawing.PointF" />.</returns>
```

[the remark this answers is not in the document]
Comment (gewarren) [reply xml/System.Drawing/PointF.xml:685]: I think my suggestion is more correct according to https://github.com/dotnet/dotnet-api-docs/wiki/Returns:-Operators.

**Comment by maryamariyan** [comment]
> Thanks. Left some suggestions for you to consider.

Awesome thanks @gewarren and @carlossanlop for the suggestions. Went through them just now.

**Pull request merged**